src/app/backend/handler/apihandler.go

```diff
@@ -381,6 +383,16 @@ func CreateHTTPAPIHandler(client *clientK8s.Clientset, heapsterClient client.Hea
 		apiV1Ws.GET("/deployment/{namespace}/{deployment}/oldreplicaset").
 			To(apiHandler.handleGetDeploymentOldReplicaSets).
 			Writes(replicasetlist.ReplicaSetList{}))
+
+	apiV1Ws.Route(
+		apiV1Ws.PUT("/scale/{kind}/{namespace}/{name}/{count}").
+			To(apiHandler.handleUpdateReplicaCount).
+			Writes(scale.ReplicaCounts{}))
+	apiV1Ws.Route(
+		apiV1Ws.GET("/scale/{kind}/{namespace}/{name}").
```

Comment:
I like, that it's generic and should be able to handle all of scalable resources.